desktop/core/src/desktop/js/reactComponents/FileChooser/FileSystemList/FileSystemList.scss

```diff
@@ -0,0 +1,38 @@
+// Licensed to Cloudera, Inc. under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  Cloudera, Inc. licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+@import '../../../components/styles/colors';
+
+.file-system__panel.ant-menu {
+  height: 554px;
+  width: 187px;
+  background-color: $fluid-gray-100;
+  padding-top: 20px;
+}
+
+.ant-menu-item {
```

Comment:
If these are styles that we need to have globally applied to all ant menus then we should move ant-menu-item, ant-menu-item-icon and ant-menu-item-selected to the file `root-wrapped-antd.less`. If they are local to this component we should contain the changes by wrapping them or if possible using our own classes instead.